addons/project/data/mail_template_data.xml

```diff
@@ -56,26 +56,26 @@
             </t>
         </td></tr>
         <tr><td style="text-align: center;">
-            <table border="0" cellpadding="0" cellspacing="0" width="590" summary="o_mail_notification" style="width:100%; margin: 32px 0px 32px 0px;">
-                <tr><td style="font-size: 13px;">
+            <table border="0" cellpadding="0" cellspacing="0" width="590" summary="o_mail_notification" style="width:100%; margin: 32px 0px 32px 0px; display: inline-table;">
+                <tr><td style="font-size: 13px;text-align:center;">
```

Comment:
Hello, @xavierbol done it for achieving below two point:

- give an accurate overview of what the customer will be receiving while preventing users from rating themselves:
  - https://nimb.ws/tHiebE chatter > if possible, display the emojis of the rating email template but disabled
- https://i.imgur.com/z7vN7vP.png the emojis in the rating email template should be centered

  
 as well as merged of this commit:
 https://github.com/odoo/odoo/commit/7710c3331ebd22f8396870bd0731f8c1152d9c41#diff-f7554252ccc99eb8b7ba19b2e804a740312310bcd0fdc009445e875c17ecb995R9
 
 'o_mail_notification' display property is none. that's why adding display: inline-table

Reply:
@kais-odoo actually, why not removing `summary="o_mail_notification"` that class is really useful?

Reply:
> @kais-odoo actually, why not removing `summary="o_mail_notification"` that class is really useful?

Agree. Because we use ` display: inline-table;` inline css, we don't need this. I have removed it.